x/axelarnet/keeper/msg_server.go

```diff
@@ -196,37 +195,10 @@ func (s msgServer) ExecutePendingTransfers(c context.Context, _ *types.ExecutePe
 			continue
 		}
 
-		chain, ok := s.GetCosmosChainByAsset(ctx, pendingTransfer.Asset.Denom)
-		if !ok {
-			s.Logger(ctx).Error(fmt.Sprintf("no cosmos chain found for asset '%s'", pendingTransfer.Asset.Denom))
-			continue
-		}
-
-		asset, ok := s.GetAsset(ctx, chain.Name, pendingTransfer.Asset.Denom)
-		if !ok {
-			s.Logger(ctx).Error(fmt.Sprintf("asset %s not found for chain '%s'", pendingTransfer.Asset.Denom, chain.Name))
-			continue
-		}
-
-		if pendingTransfer.Asset.Amount.LTE(asset.MinAmount) {
-			s.Logger(ctx).Debug(fmt.Sprintf("skipping deposit from recipient %s due to deposited amount being below minimum amount", pendingTransfer.Recipient.Address))
-			continue
-		}
-
```

Comment:
how do we ensure that we do not do the transfer if its below the minimum amount?

Reply:
it get filtered in `GetTransfersForChain`